### PR TITLE
update router config

### PR DESCRIPTION
**Pull request by festinalente91**
- updated react-native-router-flux config to use constants.

### CODE REVIEW

File: shared/containers/allLayout.js [hunk 1]
```diff
@@ -113,12 +149,15 @@ class All extends Component {
...
-          <HeaderLayout/>
+        <HeaderLayout/>
         <View style={{flex: 1}}>
           <Router
-            scenes={scenes}
             backAndroidHandler={() => this.backAndroidHandler()}
-          />
+          >
+            <Scene key={SCENE_KEY.ROOT} hideNavBar={true}>
+              {SCENE_PROPS.map(props => <Scene {...props}/>)}
```
Comment (ochanje210) [inline shared/containers/allLayout.js]: wow thank you. this is so much better! btw have you tested on this?

Comment (festinalente91) [inline shared/containers/allLayout.js]: on the virtual device :)

Comment (ochanje210) [inline shared/containers/allLayout.js]: great

**Comment by ochanje210** [comment]
shipit

**Pull request merged**